### PR TITLE
remove deprecated infobools

### PR DESCRIPTION
**Pull request by ronie**
remove deprecated stringcompare infobools.

### CODE REVIEW

**Comment by ronie** [comment]
jenkins build this please

**Review by xhaggi — APPROVED** [review]
Looks good

**Comment by ksooo** [comment]
This PR seems to cause serious trouble: http://forum.kodi.tv/showthread.php?tid=298461&pid=2477340#pid2477340

**Comment by BigNoid** [comment]
Estuary still has some deprecated infobools: https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/DialogSeekBar.xml#L3
https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/Custom_1109_TopBarOverlay.xml#L29
https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/Variables.xml#L190

Will do a PR in a minute

**Comment by AlwinEsch** [comment]
This breaks also the skins on pvr.vdr.vnsi.

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -5254,49 +5254,6 @@ int CGUIInfoManager::TranslateSingleString(const std::string &strCondition, bool
       return SYSTEM_ALWAYS_FALSE;
     else if (cat.name == "true" || cat.name == "yes")
       return SYSTEM_ALWAYS_TRUE;
-
-    // deprecated begin
-    // should be removed before L*** v18
-    if (cat.name == "isempty" && cat.num_params() == 1)
-      return AddMultiInfo(GUIInfo(STRING_IS_EMPTY, TranslateSingleString(cat.param(), listItemDependent)));
-    else if (cat.name == "stringcompare" && cat.num_params() == 2)
```
Comment (AlwinEsch) [inline xbmc/GUIInfoManager.cpp]: This is used on pvr.vdr.vnsi, is there a alternative for a `<visible>StringCompare(Window.Property(menu),osd)</visible>
`

**Comment by ronie** [comment]
then those skins need to be updated as well ;-)

you need to use String.IsEqual() in that case

**Comment by AlwinEsch** [comment]
Thanks a lot! :D

**Comment by enen92** [comment]
@ronie found this comment while going through the infolabels docs:
https://github.com/xbmc/xbmc/blob/master/xbmc/GUIInfoManager.cpp#L6716

Since `IntegerGreaterThan` has been removed, is this still usefull?

**Comment by ronie** [comment]
IntegerGreaterThan() was replaced by Integer.IsGreater(), so i guess the code is still useful..

**Comment by enen92** [comment]
True. Sorry for the noise. I'll update the comment to refer to the correct infobool and avoid future confusion

**Pull request merged**